version.py

```diff
@@ -11,4 +11,4 @@
 # Typically, adding a dependency only requires a minor version bump, and
 # removing a dependency requires a major version bump.
 
-PROVISION_VERSION = '29.1'
+PROVISION_VERSION = '29.3'
```

Comment:
When new dependency is added or removed, then only we require to change `PROVISION_VERSION `. In your case, it is not required. So you can remove the changes here.

Reply:
@YJDave Okay, but Rishi Gupta told me to do so,  that why I did, but the problem here is that I have not rights to do so due to access permission (github) 
Do you want me to remove provision number commit?

Reply:
Can you link the discussion/message here where Rishi suggests you? That'll be helpful.

Reply:
you can here https://github.com/zulip/zulip/pull/11950#issuecomment-479334950

Reply:
though i removed update provision version commit, it always conflicts, if you think it is necessary then please do it by yourself because i don't have permission to do that.

Reply:
Sorry for the confusion. I believe you need to change version number and then force push the changes. I am not able to understand that "you don't have permission". So please wait for others to comment on that.

Reply:
Let me try one more time